### ISSUE TITLE
Performance: Loop selected steps as soon as 1+ step button is being held -- possible?

### ISSUE THREAD

**Issue by jbjhjm** [westlicht/performer#350]
Hi everyone! There's one neat feature known by some other sequencers that doesn't seem to exist, or I was just unable to find it. In that case correct me please.

Is there a performance feature that will only loop the selected steps as soon as some step buttons are pressed?
This is pretty cool for adding a bit of life and fills to sequences.
Even looping a single step by pressing it's single button a few times can yield interesting results.
For example Varigate/Voltage block can do so.

Apart from these few small extra features I'm truely blown away by range of functions and features as well as how accessible they are. A huge thank you and much respect for your tech skills @ westlicht!

**Comment by Lokua** [comment]
I don't think this is possible to do from any of the main sequencing pages without major breaking changes to existing, critical functions. When you select 2 buttons (that would represent a range in your request) on the gate page, that toggles their state to or from ON/OFF, so it couldn't be done there. Likewise, if you do this on any other step sequencing parameter page, holding 2 steps enables it so you can edit both at the same time. I don't see how this feature can co-exist within the STEPS context without a third button involved, and the options for those can only be the buttons on the left or the buttons under the screen, which for hopefully obvious reasons cannot be used as they too represent critical functions to the current workflow.

The only way I can think of how to pull this off in a non-breaking and relatively intuitive way would be to purpose one of the unused "pages", for example pressing PAGE+T6 could theoretically enter a new page where one could quickly set the start/end. I imagine it could function exactly like the FIRST and LAST modes, only you press 2 buttons instead of one, then release the PAGE button to exit.

One drawback to this of course is that this mode wouldn't be represented on any existing faceplates. Not really an issue for me.

I think it's a cool idea for sure.
